glang.spec

```diff
@@ -1,5 +1,5 @@
 Name:           glang
-Version:        0.2.1
+Version:        0.3.0
```

Comment:
Can you update this version as well please
https://github.com/DaveDuck321/GrapheneLang/blob/da3b7d5e7c8289045bd2beb8ea425daf5aa098f4/pyproject.toml#L8

One day we'll have a single version number...

Reply:
Thanks :blush:. I knew I was forgetting something